Adding Deprecation Note for Prereq Tool (#315)
IBM/cp4waiops-samples: prereq-checker/4.10/Readme.md

# IBM Cloud Pak for AIOps prerequisite checker tool

> **Note: For IBM Cloud Pak for AIOps v4.10.1+, the prerequisite checker tool script has been deprecated and is no longer supported. To check prerequisites for CP4AIOPS, please use the aiopsctl plugin. For more info, please refer to the offical IBM Docs regarding CP4AIOPs installation [here](https://ibm.biz/cp4aiops_410_install).**

Reviewer: ```suggestion
**Note: As of IBM Cloud Pak for AIOps v4.10.1, the prerequisite checker tool script is no longer supported. To check prerequisites for CP4AIOps, please use the aiopsctl plugin. For more info, please refer to the [IBM Documentation](https://ibm.biz/cp4aiops_410_install).**
```
